WeakAuras/Types.lua

```diff
@@ -1589,8 +1597,6 @@ WeakAuras.internal_fields = {
   parent = true,
   internalVersion = true,
   regionType = true,
```

Comment:
regionType is probably also problemeatic.

Reply:
Ah, you're probably right on that one. I think Display would be the best category for regionType, given just how much of the display tab data is governed by the regionType.

Reply:
I agree

Reply:
on further thought, sortHybridTable (and much of the dynamic group options such as growth direction) should be categorized as arrangement